voorbeeld-styled-comp/src/app/components/pokemon/Voorbeeld.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+interface Props {
+  voornaam: string;
+  achternaam: string;
+  leeftijd: number;
+}
+
+const Voorbeeld: React.FC<Props> = ({ voornaam, achternaam, leeftijd }) => {
+  return (
+    <div>
+      <h1>{voornaam}</h1>
+      <h1>{achternaam}</h1>
+      <h1>{leeftijd}</h1>
+    </div>
+  );
```

Comment:
Dit is echt slecht.

Waarom maak je er niet {} van?

Reply:
Neh.